migrations/2023-06-21-153242_add_captcha/up.sql

```diff
@@ -0,0 +1,5 @@
+create table captcha_answer (
+    uuid uuid not null primary key,
```

Comment:
1. You should be able to use the `gen_random_uuid ()` function to create a default one, without having to generate it in code. https://stackoverflow.com/a/33274393
2. Add published and updated columns like every other table. Published is what you will use to delete old captchas. You can leave off updated if you choose, since it won't be used, up to you.
3. I'd prefer you still used auto-incrementing ids like every other table, and then included the uuid as a 2nd but unique column. iirc making a column unique automatically makes it perform the same speed as an index. Ultimately your call tho.

Reply:
should all be done, i've left off updated and added a serial id

Reply:
@dessalines Whats the point of having an int id column in this case? Its not used anywhere, and uuid works just fine as primary key.